File: Homify.Angular/src/backend/services/device/models/CreateCameraRequest.ts
export default interface CreateCameraRequest {
  name: string;
  model: string;
  isExterior: boolean;
  isInterior: boolean;
	movementDetection: boolean;
	peopleDetection: boolean;
  description: string;
  photos: Array<string>;
  ppalPicture: string;
}

File: Homify.WebApi/Controllers/Devices/Models/Requests/CreateCameraRequest.cs
namespace Homify.WebApi.Controllers.Devices.Models.Requests;

public sealed record class CreateCameraRequest
{
    public string? Name { get; init; }
    public string? Model { get; init; }
    public bool? IsExterior { get; init; }
    public bool? IsInterior { get; init; }
    public bool? MovementDetection { get; init; }
    public bool? PeopleDetection { get; init; }
    public string? Description { get; init; }
    public List<string>? Photos { get; init; }
    public string? PpalPicture { get; init; }
}
